src/shapes/line.py

```diff
@@ -0,0 +1,86 @@
+import cv2
+import numpy as np
+from numpy import ndarray
+
+from src.shapes.point import Point
+from src.shapes.shape import Shape
+from src.transform_shapes.transform_shapes import TransformShape
+
+
+class Line(Shape, TransformShape):
+    """
+    A class representing a line shape.
+    """
+
+    def __init__(self, point1: Point, point2: Point, line_color: tuple[float, float, float]):
```

Comment:
move linecolor to base

Reply:
I tried to make the base shape as interface not as abstract class.
Traditional interface is only for methods